packages/nimble-components/src/rich-text/models/markdown-serializer.ts

```diff
@@ -35,13 +50,21 @@ export class RichTextMarkdownSerializer {
             });
         };
 
+        const mentionNode = function mention(
+            state: MarkdownSerializerState,
+            node: Node
+        ): void {
+            const href = node.attrs.href as string;
+            state.write(`<${href}>`);
```

Comment:
While looking into the `prosemirror-markdown`, there is a method named `esc(str: string, startOfLine = false)` which will be escaping the special markdown stuffs. 

https://github.com/ProseMirror/prosemirror-markdown/blob/3e5a5d02f7c1b9336744740149767ee1dc4e954d/src/to_markdown.ts#L405C2-L416C4. 

Kindly share your thoughts on this.

Reply:
The built-in auto-link feature itself does not escape the link. Also, we aren't able to use the auto-link serializer because the `link` is mark but in tiptap `mention` is a node. So we can't use the existing logic. Instead, we wrote our logic which is very similar to the absolute link serialization. 

Added the reference code as you suggested.